ci-operator/step-registry/baremetal/lab/nfs-provisioner/baremetal-lab-nfs-provisioner-ref.yaml

```diff
@@ -18,6 +18,12 @@ ref:
         The NFS_SERVER that exports a share to be used by the node. It's set by default to the same value as the AUX_HOST. 
         The NFS_SERVER must exports the /opt/nfs/${CLUSTER_NAME} folder, to be consumed by the provisioner, and 
         assumed as already existing.
+    - name: NFS_CLIENT_MIRRORED_IMAGE_URL
+      default: "quay.io/openshifttest/nfs-subdir-external-provisioner@sha256:3036bf6b741cdee4caf8fc30bccd049afdf662e08a52f2e6ae47b75ef52a40ac"
```

Comment:
No need to add it here as we don't expect (and we don't want actually) users to change the value from the test configs. A variable in the script and a comment above it is enough.